doc/contributor-how-to/contribute-layers.md

```diff
@@ -28,6 +39,14 @@ group.
   qgreenland/config/layers/Group/Subgroup/new_layer.py
 ```
 
+Once the new_layer.py file is generated, follow the documentation within the file
+to fill out your layer configuration.
+
+```{note}
+If the group directory where you have created your layer file has a settings.py file,
```

Comment:
- [] "settings.py" --> `__settings__.py` :)

Reply:
Maybe link to the reference documentation for layer groups? https://qgreenland.readthedocs.io/en/latest/reference/api/models.html#module-qgreenland.models.config.layer_group